hr_attendance_modification_tracking/tests/test_hr_attendance.py

```diff
@@ -0,0 +1,108 @@
+        # Use case 1:
+        # Create an attendance with correct (now) check-in and leave it open
+        # Expected: manually_changed = False
+        self.employee_01 = self.env["hr.employee"].create({"name": "Employee01"})
+        dti = datetime.now()
+        att = self.hr_attendance.create(
+            {"employee_id": self.employee_01.id, "check_in": dti.strftime(DF)}
+        )
+        self.assertEqual(
+            att.time_changed_manually, False, "Use case 1: Wrong value, not changes."
+        )
+        # Use case 2:
+        # Create an attendance with incorrect (now - 10 minutes) check-in and
+        # leave it open. Maximum tolerance is one minute from now.
+        # Expected: manually_changed = True
+        self.employee_02 = self.env["hr.employee"].create({"name": "Employee02"})
+        dti = datetime.now() - relativedelta(minutes=10)
+        att = self.hr_attendance.create(
+            {"employee_id": self.employee_02.id, "check_in": dti.strftime(DF)}
+        )
+        self.assertEqual(
+            att.time_changed_manually,
+            True,
+            "Use case 2: Wrong value, tolerance exceeded",
+        )
+        # Use case 3:
+        # Create an attendance with incorrect (now + 10 minutes) check-in
+        # and leave it open.Maximum tolerance is one minute from now.
+        # Expected: manually_changed = True
+        self.employee_03 = self.env["hr.employee"].create({"name": "Employee03"})
+        dti = datetime.now() + relativedelta(minutes=10)
+        att = self.hr_attendance.create(
+            {"employee_id": self.employee_03.id, "check_in": dti.strftime(DF)}
+        )
+        self.assertEqual(
+            att.time_changed_manually,
+            True,
+            "Use case 3: Wrong value, tolerance exceeded.",
+        )
+        # Use case 4:
+        # Create an attendance with correct (now - 15 sec) check-in and correct
+        # (now + 15 sec) check-out.
+        # Expected: manually_changed = False
+        self.employee_04 = self.env["hr.employee"].create({"name": "Employee04"})
+        dti = datetime.now() - relativedelta(seconds=15)
+        dto = datetime.now() + relativedelta(seconds=15)
+        att = self.hr_attendance.create(
+            {
+                "employee_id": self.employee_04.id,
+                "check_in": dti.strftime(DF),
+                "check_out": dto.strftime(DF),
+            }
+        )
+        self.assertEqual(
+            att.time_changed_manually,
+            False,
+            "Use case 4: Wrong value, tolerance not exceeded.",
+        )
+        # Use case 5:
+        # Change previous attendance check-out to now + 1 hour
+        # Expected: manually_changed = True
+        dto = datetime.now() + relativedelta(hours=1)
+        att.write({"check_out": dto.strftime(DF)})
+        self.assertEqual(
+            att.time_changed_manually,
+            False,
+            "Use case 5: Wrong value, data changed.",
```

Comment:
I don't think changing this assert from True to False is something valid. This shows the behavour has been changed in the migration.

Reply:
What do you think @Saran440 ??